### DISCUSSION TITLE
Qtile Wayland Problems

### DISCUSSION THREAD

**Discussion by offcobra** [qtile/qtile#4276] [Q&A]
Does anyone get an amdgpu_cs_ctx_create2 failed error on starting qtile Wayland?

I'm starting it from tty with: qtile start -b wayland

I have all the dependencies installed. Qtile check has no error in output.

Sway / Hyprland work just fine...

What am I doing wrong? Is it a Qtile bug or an AMD driver bug?
Does it have something to do with the WAYLAND_DISPLAY env Variable?

As a DevOps Engineer with Python Development background, this is really frustrating for me...

Os: ArcoLinux

PS: I use Qtile X11 as a daily driver (on every PC, even at work...) and i would like to switch to wayland.

**Comment by e-v-o-l-v-e** [comment]
hey, more than one year later i have the exact same problem, though with gnome or kde or wayfire, only hyprland works, i'm on nixos, did you manage to solve the problem ?
